app/lib/content_security_policy.rb

```diff
@@ -72,6 +72,14 @@ def s3_hostname_host
     host_to_url ENV.fetch('S3_HOSTNAME', nil)
   end
 
+  def swift_object_url
+    url = ENV.fetch('SWIFT_OBJECT_URL', nil)
+    unless url =~ %r!/$!
+      url += "/"
+    end
```

Comment:
I think the following would be clearer
```suggestion
    url += '/' unless url.end_with?('/')
```